modules/shortcodes/js/slideshow-shortcode.js

```diff
@@ -143,6 +143,23 @@ JetpackSlideshow.prototype.renderControls_ = function() {
 		var controlName = controls[ i ];
 		var a = document.createElement( 'a' );
 		a.href = '#';
+		switch ( i ) {
+			case 0:
+				a.className = 'classic-jetpack-slideshow_button-prev';
+				a.setAttribute( 'aria-label', 'Previous Slide' );
```

Comment:
I think we'll need to translate this. You could move those strings here:
https://github.com/Automattic/jetpack/blob/89a9af96b669e2e5a2ed47d3f3e07c804d6e0dd0/modules/shortcodes/slideshow.php#L289

And then call them in the js file with `jetpackSlideshowSettings`.

Reply:
This is done.